DuckDuckGo/BrowsingMenu/BrowsingMenuAnimator.swift

```diff
@@ -61,7 +61,7 @@ final class BrowsingMenuAnimator: NSObject, UIViewControllerAnimatedTransitionin
             transitionContext.containerView.addSubview(fromSnapshot)
         }
 
-        toViewController.bindConstraints(to: fromViewController.currentTab?.webView)
+        toViewController.bindConstraints(to: fromViewController.viewCoordinator.contentContainer)
```

Comment:
the web view can now be as big as the app so is the wrong thing to base the browsing menu on